crt_portal/cts_forms/management/commands/update_response_templates.py

```diff
@@ -10,6 +10,13 @@ class Command(BaseCommand):  # pragma: no cover
     help = 'Adds new response templates or updates existing ones'
 
     templates_dir = os.path.join(settings.BASE_DIR, 'cts_forms', 'response_templates')
+    template_ids = []
+
+    def template_exists(self, object):
+        if object.title in self.template_ids:
+            return
+        else:
+            return object
```

Comment:
I think this can be removed now - looks like it's not used since the refactor below?

```suggestion
```